gorm/collection_operators.go

```diff
@@ -1,21 +1,24 @@
 package gorm
 
 import (
+	"context"
 	"fmt"
 	"strings"
 
-	"github.com/infobloxopen/atlas-app-toolkit/query"
+	"github.com/golang/protobuf/proto"
 	"github.com/jinzhu/gorm"
+
+	"github.com/infobloxopen/atlas-app-toolkit/query"
 )
 
 // ApplyCollectionOperators applies collection operators to gorm instance db.
-func ApplyCollectionOperators(db *gorm.DB, obj interface{}, f *query.Filtering, s *query.Sorting, p *query.Pagination, fs *query.FieldSelection) (*gorm.DB, error) {
-	db, fAssocToJoin, err := ApplyFiltering(db, f, obj)
+func ApplyCollectionOperators(ctx context.Context, db *gorm.DB, obj interface{}, pb proto.Message, f *query.Filtering, s *query.Sorting, p *query.Pagination, fs *query.FieldSelection) (*gorm.DB, error) {
```

Comment:
Is it not a breaking change ? we recently removed `ctx` from here because its anti pattern